core-bundle/src/File/SubtitleType.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\File;
+
+enum SubtitleType
```

Comment:
These need to be string backed in order to be used as value for the database. (The names need to stay independent of the data.)

Reply:
See 79379327d064375187b5495040d1aba849018ec9